app/main.py

```diff
@@ -1,8 +1,45 @@
 class Car:
-    # write your code here
-    pass
+
+    def __init__(self, comfort_class: int,
+                 clean_mark: int, brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
```

Comment:
Move each parameter, including self, to a new line is a recommended practice for improving code readability,

Reply:
fix in all places

Reply:
ok. code  has already fixed. please check it again